modules/localstack/src/test/java/org/testcontainers/containers/localstack/SimpleLocalstackTest.java

```diff
@@ -47,6 +56,17 @@ public void s3Test() throws IOException {
         final S3Object object = s3.getObject("foo", "bar");
         final String content = IOUtils.toString(object.getObjectContent(), Charset.forName("UTF-8"));
         assertEquals("The object can be retrieved", "baz", content);
+    }
+
+    @Test
+    public void hostnameSqsTest() {
```

Comment:
It would be nice if we could make this test verify correct end-to-end behaviour rather than just check that the URL reflects the provided hostname.

Is there an easy to test scenario where the external hostname matters?

Reply:
Have added some tests using AWS CLI.